src/System.Net.NetworkInformation/tests/FunctionalTests/NetworkChangeTest.cs

```diff
@@ -27,5 +27,25 @@ private void NetworkChange_NetworkAddressChanged(object sender, EventArgs e)
         {
             throw new NotImplementedException();
         }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/308
+        public void NetworkAvailabilityChanged_AddRemove_Success()
+        {
+            NetworkAvailabilityChangedEventHandler handler = NetworkChange_NetworkAvailabilityChanged;
+            NetworkChange.NetworkAvailabilityChanged += handler;
+            NetworkChange.NetworkAvailabilityChanged -= handler;
+        }
+
+        [Fact]
+        public void NetworkAvailabilityChanged_JustRemove_Success()
+        {
+            NetworkAvailabilityChangedEventHandler handler = NetworkChange_NetworkAvailabilityChanged;
+            NetworkChange.NetworkAvailabilityChanged -= handler;
+        }
+
+        private void NetworkChange_NetworkAvailabilityChanged(object sender, NetworkAvailabilityEventArgs e)
+        {
+            throw new NotImplementedException();
```

Comment:
Yeah, it actually would. But interestingly, the same should be true for the existing tests in this class, which should fail if any interface addresses are added while the event is subscribed, and I've never seen such a failure... I'll remove both of the exceptions.